### ISSUE TITLE
Media queries

### ISSUE THREAD

**Issue by Andrey-1992** [Andrey-1992/spacestagram-challenge#9]
As a developer, I should be able to create a responsive app for:
- Desktop view
- Tablet view (portrait/landscape)
- Mobile view (portrait/landscape)
